docs/chapters/sdk/run-binary-on-target.rst

```diff
@@ -0,0 +1,21 @@
+Deploying and running binary on target
+======================================
+
+Once the binary has been cross compiled it needs to be installed on the target. To do so the easiest way is to use SSH. For that to work you need to know the IP address of your target.
+
+.. note:: The `Raspberry Pi documentation`_ describes how to find a new device on the network. Finding the IP address for our device will work in the same manner.
+
+.. _`Raspberry Pi documentation`: https://www.raspberrypi.org/documentation/remote-access/ip-address.md
+
+The default user and at the same time password on the PELUX image is `root`. Use `scp` to copy your binary to the target.
```

Comment:
PELUX image default user and password is `root`. 

Question: Is this true? Isn't it just root and no password?